### ISSUE TITLE
Navigate to next entry in "Find in Files..." and open in editor keyboard shortcut (F4)

### ISSUE THREAD

**Issue by hartmut27** [notepadqq/notepadqq#489]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[X] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/notepadqq/notepadqq/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Find in Files, navigate to next finding, there is not yet a Keyboard Shortcut for this

**Expected behavior**
Find in Files, navigate to next place where something was found, there is a Keyboard Shortcut F4 for invocation. User is repeating F4 to go to next reference, and so forth. This goes down in the list of find results, and at the same time, opens editor at this specific location (the same as when I double click on the find entry).

**Please tell us about your environment:**
Ubuntu 17.04

**Comment by JuBan1** [comment]
Will be fixed by #446 once we get it merged.

**Comment by JuBan1** [comment]
Fixed by de15c09
